lms/templates/student_account/form_field.underscore

```diff
@@ -40,7 +40,9 @@
            type="<%= type %>"
            name="<%= name %>"
            class="input-block <% if ( type === 'checkbox' ) { %>checkbox<% } %>"
-           aria-describedby="<%= form %>-<%= name %>-desc"
+            <% if ( type !== 'password' ) { %>
+            aria-describedby="<%= form %>-<%= name %>-desc"
+            <% } %>
```

Comment:
On second thoughts, I think making this one line like the if conditions below would result in cleaner html. :)